src/main/ConfigLoader.ts

```diff
@@ -29,31 +34,33 @@ async function loadFileWithName(
     loaders: Array<IFileLoader>,
     configPath: string,
     name: string,
-): Promise<Array<object>> {
```

Comment:
Interesting that this type is just wrong... this file returns a `Promise<object>` not a `Promise<Array<object>>`.